src/main/java/com/customerApi/api/controllers/CustomerController.java

```diff
@@ -0,0 +1,44 @@
+package com.customerApi.api.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.services.CustomerService;
+
+
+
+@Controller
+public class CustomerController {
+	private static final Logger logger=LoggerFactory.getLogger(CustomerController.class);
+	
+	@Autowired
+	CustomerService service;
+	
+	
+	@PostMapping("/register")
+	
+	public ResponseEntity <Customer> register(@RequestBody Customer data)
+	{
+		logger.info("Redirect to registration service"+data);
```

Comment:
The proper way of printing log is below, its an example
_For example, if you want to print 1 value"_
logger.info("Customer Name {}, sign-in in", customer.getName());

_For example, if you want to print 2 values or multiple"_
logger.info("Customer First Name {}, last name {}, sign-in in", customer.getFirstName(), customer.getLastName());

Reply:
I made changes to print meaningful message in logger.